src/main/scala/cromwell/engine/db/slick/JesJobComponent.scala

```diff
@@ -3,8 +3,8 @@ package cromwell.engine.db.slick
 case class JesJob
 (
   executionId: Int,
-  jesId: Int,
-  jesStatus: String,
+  jesId: Option[String],
```

Comment:
Yeah that's right- the ID from JES is definitely a string (even without the 'operations/' prefix). I guess Int was just chosen at random when the field was first created, before the actual content for it was known.